v2/go.mod

```diff
@@ -0,0 +1,19 @@
+// Copyright 2014 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module github.com/google/btree/v2
+
+go 1.18
+
+require github.com/petar/GoLLRB v0.0.0-20210522233825-ae3b015fd3e9 // indirect
```

Comment:
nit: maybe this should either be removed e.g. through moving btree_mem to a separate package.  Otherwise it is strange that v2 package requires gollrb

Reply:
btree_mem is already in a separate `main package` (or am I misunderstanding your question?). Where would you want ot move it?

Reply:
Sorry, the right term would be a separate module (basically a separate go.mod file).  This will allow to split dependencies between benchmark and the v2 module (since v2 does not depend on gollrb).